src/components/scenes/SendScene.js

```diff
@@ -107,8 +91,8 @@ type State = {
 } & WalletStates
 
 class SendComponent extends React.PureComponent<Props, State> {
-  addressTile: ?React.ElementRef<typeof AddressTile>
-  pinInput: ?React.ElementRef<typeof TextInput> = React.createRef()
+  addressTile: AddressTile | void
+  pinInput: React.ElementRef<typeof TextInput> = React.createRef()
 
   constructor(props: Props) {
     super(props)
```

Comment:
Can we destructure `guiMakeSpendInfo` out of `props.route.params` at the top of the constructor, and then use that throughout the rest of the method?